src/main/java/com/julienvey/trello/Trello.java

```diff
@@ -79,6 +79,10 @@
 
     TList getList(String listId, Argument... args);
 
+    /* CheckLists */
+
+    CheckList getCheckList(String checkListId, Argument... args);
```

Comment:
I would prefer getCardCheckList for the name

It would be consistent with other methods names such as "getCardAttachments"